mathics_django/web/controllers/doc.py

```diff
@@ -21,12 +22,37 @@
 
 DocResponse = Union[HttpResponse, JsonResponse]
 
+seen_pymathics_modules = copy(pymathics_modules)
+
 
 def check_for_pymathics_load():
+    global seen_pymathics_modules
     if seen_pymathics_modules != pymathics_modules:
-        # print("XXX refresh pymathics doc")
-        global documentation
-        documentation = MathicsDjangoDocumentation()
+        print("XXX refresh pymathics doc", pymathics_modules)
+        new_modules = pymathics_modules - seen_pymathics_modules
+        for new_module in new_modules:
+            mathics3_module_part = documentation.parts_by_slug.get(
+                "mathics3-modules", None
+            )
+            # If this is the first loaded module, we need to create the Part
+            if mathics3_module_part is None:
+                mathics3_module_part = self.doc_part(
```

Comment:
`self` is not right, it is `documentation` here. Must be a cut and paste error.

Reply:
I have incorporated the good parts of this PR into #202

Reply:
Indeed, it was a cut-and-paste error.